lumicks/pylake/file.py

```diff
@@ -104,6 +111,8 @@ def print_dataset(dset, name, indent):
             r += f"{space}- Size: {dset.size}\n"
             return r
 
+        blacklist = self.blacklist
```

Comment:
I don't think that this is needed. You can just use `self.blacklist` in the single use of the variable below.

Reply:
Yeah. 👍